<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1800</width>
    <height>850</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1800</width>
    <height>850</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1800</width>
    <height>850</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1800</width>
     <height>850</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1800</width>
     <height>850</height>
    </size>
   </property>
   <widget class="QLabel" name="tytul_gl">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:700;&quot;&gt;USTAWIENIA REGULATORA PID:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="wej_sym">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;USTAWIENIA GENERATORA:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="ksztalt">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>630</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;KSZTAŁT GENERATORA:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>670</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="sin">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Sinusoida</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="prost">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Prostokąt</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="krok_jedn">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Skok jednostkowy</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>790</y>
      <width>421</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QHBoxLayout" name="Buttons_layout">
     <item>
      <widget class="QPushButton" name="Start_Button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>START</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Reset_Button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>RESET</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Stop_Button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>STOP</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>450</y>
      <width>341</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Amplituda:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QDoubleSpinBox" name="wej_T"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Okres:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QDoubleSpinBox" name="wej_t">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Chwila aktywacji:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QDoubleSpinBox" name="wej_Amp"/>
     </item>
     <item row="3" column="2">
      <widget class="QDoubleSpinBox" name="wej_p"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Wypełnienie:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="emp5_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="emp5_3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="emp5_4">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="emp5_5">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget1" native="true">
    <property name="geometry">
     <rect>
      <x>469</x>
      <y>0</y>
      <width>1321</width>
      <height>840</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>411</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="2">
      <widget class="QDoubleSpinBox" name="Aa">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="emp5">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QDoubleSpinBox" name="Ba">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="wektorA">
       <property name="text">
        <string>Współczynniki wektora A:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QDoubleSpinBox" name="Ac">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QDoubleSpinBox" name="Ab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QDoubleSpinBox" name="Bb">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QDoubleSpinBox" name="Bc">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="wektorB">
       <property name="text">
        <string>Współczynniki wektora B:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="emp4">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Opóźnienie:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="emp5_6">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QDoubleSpinBox" name="opoznienie"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>411</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="1" column="0">
      <widget class="QLabel" name="rozniczkowanie">
       <property name="text">
        <string>Stała różniczkowania:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="emp2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="calkowanie">
       <property name="text">
        <string>Stała całkowania:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="emp1">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QDoubleSpinBox" name="st_calk">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QDoubleSpinBox" name="st_rozn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="wzmocnienie">
       <property name="text">
        <string>Wzmocnienie:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="emp3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QDoubleSpinBox" name="wzmocn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="wej_sym_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;USTAWIENIA MODELU ARX:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>740</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="Conf_Button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>ZMIEŃ KONFIGURACJE</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="Single_shot_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>SKOK JEDNOSTKOWY</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sin</sender>
   <signal>clicked(bool)</signal>
   <receiver>prost</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>499</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>503</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sin</sender>
   <signal>clicked(bool)</signal>
   <receiver>krok_jedn</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>497</y>
    </hint>
    <hint type="destinationlabel">
     <x>595</x>
     <y>496</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>prost</sender>
   <signal>clicked(bool)</signal>
   <receiver>sin</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>469</x>
     <y>494</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>499</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>prost</sender>
   <signal>clicked(bool)</signal>
   <receiver>krok_jedn</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>527</x>
     <y>495</y>
    </hint>
    <hint type="destinationlabel">
     <x>628</x>
     <y>490</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>krok_jedn</sender>
   <signal>clicked(bool)</signal>
   <receiver>sin</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>663</x>
     <y>494</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>494</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>krok_jedn</sender>
   <signal>clicked(bool)</signal>
   <receiver>prost</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>668</x>
     <y>496</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>496</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
